src/authentication/accessTokenRefresher.ts

```diff
@@ -21,14 +21,12 @@ export class AccessTokenRefrsher {
     private async refreshToken(): Promise<void> {
         const settings = await this.settingsProvider.getSettings();
 
-        let managementApiUrl = settings[Constants.SettingNames.managementApiUrl];
+        let backendUrl = Utils.getDeveloperEndpoint(settings[Constants.SettingNames.backendUrl]);
```

Comment:
Maybe rename `Utils.getDeveloperEndpoint` to `Utils.getBackendUrl` to match setting name?